packages/styles/scss/components/filter-panel/_filter-panel.scss

```diff
@@ -0,0 +1,426 @@
+      }
+    }
+    .#{$prefix}--input_container-heading {
+      &:focus-visible {
+        outline: $spacing-01 solid $focus;
+        margin-right: $spacing-01;
+        margin-left: $spacing-01;
+        padding-left: calc((#{$spacing-04} + #{($spacing-01)}));
+      }
+    }
+  }
+  :host(#{$dds-prefix}-input-select-item) {
+    @include use-carbon-productive-tokens();
+    @include carbon--type-style('body-short-01');
+
+    background-color: $ui-01;
+    width: 100%;
+    cursor: pointer;
+    position: relative;
+    color: $text-02;
+    padding: $spacing-03 $spacing-06 $spacing-03 $spacing-06;
+
+    &:hover {
+      background-color: $hover-ui;
+      color: $text-01;
+    }
+    &:active {
+      background-color: $highlight;
+    }
+
+    &[selected] {
+      background-color: $highlight;
+      color: $text-01;
+    }
+    .#{$prefix}--close_icon {
+      position: absolute;
+      right: $spacing-05;
+      top: calc((#{$spacing-04} - #{($spacing-01)}));
+    }
+  }
+  :host(#{$dds-prefix}-filter-panel-heading) {
+    @include use-carbon-productive-tokens();
+    @include carbon--type-style('heading-01');
+    @include carbon--breakpoint('sm') {
+      margin-left: $spacing-05;
+      margin-bottom: $spacing-05;
+    }
+    @include carbon--breakpoint('lg') {
+      margin: 0;
+    }
+
+    @include carbon--breakpoint-down('lg') {
+      display: none;
+    }
+  }
+
+  :host(#{$dds-prefix}-filter-group:first-of-type) {
+    ::slotted(#{$prefix}-filter-group-item:first-of-type) {
+      @include carbon--breakpoint-down('md') {
+        margin-top: 1rem;
+      }
+    }
+  }
+
+  :host(#{$dds-prefix}-filter-group) {
+    ::slotted(#{$dds-prefix}-filter-group-item) {
+      border-top: 1px solid $ui-03;
+      border-bottom: none;
+    }
+
+    &:last-of-type {
+      ::slotted(#{$dds-prefix}-filter-group-item) {
+        border-bottom: 1px solid $ui-03;
+      }
+    }
+  }
+
+  :host(#{$dds-prefix}-filter-group-item) {
+    @extend .#{$prefix}--accordion__item;
+
+    display: block;
+    outline: none;
+
+    .#{$prefix}--accordion__heading {
+      padding-top: calc(
+        (#{$container-03} - #{map-get($body-long-01, 'line-height')}) / 2
+      );
+      display: flex;
+      align-items: center;
+    }
+
+    .#{$prefix}--accordion__title {
+      @include use-carbon-productive-tokens();
+      @include carbon--type-style('heading-01');
+    }
+  }
+
+  :host(#{$dds-prefix}-filter-group-item[open]:not([disabled])) {
+    @extend .#{$prefix}--accordion__item--active;
+
+    .#{$prefix}--accordion__content {
+      padding-left: 0;
+      padding-right: 0;
+    }
+  }
+
+  :host(#{$dds-prefix}-filter-modal-footer) {
+    z-index: 999;
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    overflow: auto;
+  }
+
+  :host(#{$dds-prefix}-filter-modal-footer-button) {
+    @extend :host(#{$prefix}-modal-footer-button);
+
+    outline: 0;
+    &[kind='tertiary'] {
+      height: calc((#{$spacing-10} + #{($spacing-01)}));
```

Comment:
Is this needed? Why would this button type have a different height?

Reply:
there's an issue where the tertiary button is missing a few bottom pixels. I think it's a carbon-web-component issue

Reply:
Hm, removing this style sets the height correctly to `64px`. Can you show me where tertiary button is displaying differently to the other types?

Reply:
<img width="572" alt="Screen Shot 2021-07-14 at 11 35 58 PM" src="https://user-images.githubusercontent.com/56688604/125725406-adfea136-1ccc-40b2-8998-4631cbfa4b38.png">

Reply:
Here's what it looks like

Reply:
So it doesn't look like there's anything wrong with that `tertiary` button. It looks like the problem is coming from the `primary` button. It's `2px` too tall, which is why the bottom outline is getting cropped when focused.

![image](https://user-images.githubusercontent.com/909118/125726096-a969e035-52f9-4873-92fe-e1e756ab8636.png)

Reply:
Ah, Ignacio and I had spoken to Olivia about some issues with those buttons. We think there are some style issues with the  buttons from `carbon-web-components`

Reply:
The styles look ok to me. The buttons in shadow DOM have different heights than the web component button. If a specific height is needed it should be set on both (right now they're 2 different heights).